### ISSUE TITLE
No CSS variable for input height

### ISSUE THREAD

**Issue by isaachinman** [airbnb/react-dates#78]
So, as per the README, the inputs can be customised via SASS variables in `css/variables.scss`. Currently, the first two lines pertain to picker width, and the rest are all colour-related. So there is no mention of height, nor any obvious way to affect it.

Actually, trying to do it without one of these variables proves somewhat difficult, as the nested heights are a bit complicated. Any advice on this?

**Comment by majapw** [comment]
Totes right! It'd be great to include css variables for these things until we move towards a props-based `react-with-styles` implementation. Right now, at Airbnb, we do these things by overriding some of the classes like so:
```
.DateInput {
  font-size: 16px;
  font-weight: normal;
  line-height: normal;
  padding: 6px;
  width: 115px;
}
```

but it seems like it might be better to set these things as variables. We'd def accept a PR for this! :)

**Comment by isaachinman** [comment]
A variable would for sure be preferable, because (if I recall correctly from setting this up on my own project), the height, unlike the width, of these inputs is never explicitly declared, but instead generated by at least two different elements' padding.

So I think the padding will have to be generated as some ratio of the variable, haven't worked it out yet. Will try to get a PR done in the next couple of days.

**Comment by JemarJones** [comment]
What is the progress of this? I can work on a PR for this if its something you'd accept!

**Comment by isaachinman** [comment]
@JemarJones I never started on this, so go for it!

**Comment by isaachinman** [comment]
@JemarJones Do you have any interest in pursuing this?

**Comment by JemarJones** [comment]
I ended up using the method @majapw had mentioned (good enough for Airbnb === good enough for us), so it's not a current priority of mine. So i may do this at some point, but if you or someone else has time before then, thats cool too!

**Comment by isaachinman** [comment]
That doesn't really make sense, what @majapw was suggesting is just overriding existing CSS - while that might suffice in your individual use case, it shouldn't be the recommended methodology for `react-dates` on a whole.

Moreover, the styling, especially colour scheme, of `react-dates` is quite complicated, and it'd be a lot easier to swap in a new palette if these things had var names. I will work on a PR if @majapw will accept it.

**Comment by JemarJones** [comment]
Right, i'm just saying that for my use case i was okay with just overriding the css. I didn't say anything about recommending that for others, just that it's not a priority for me at the moment. But yea, go for it man.

**Comment by Andrew-Kang-G** [comment]
> Totes right! It'd be great to include css variables for these things until we move towards a props-based `react-with-styles` implementation. Right now, at Airbnb, we do these things by overriding some of the classes like so:
> 
> ```
> .DateInput {
>   font-size: 16px;
>   font-weight: normal;
>   line-height: normal;
>   padding: 6px;
>   width: 115px;
> }
> ```
> 
> but it seems like it might be better to set these things as variables. We'd def accept a PR for this! :)

For the font-family of the input box, I have found _input to be added. 
```
.DateInput_input {
  font-size: 12px;
  font-weight: bold;
}
```
